x-pack/plugins/monitoring/server/kibana_monitoring/collectors/get_kibana_usage_collector.js

```diff
@@ -60,6 +61,20 @@ export function getKibanaUsageCollector(server) {
           }
         }), {})
       };
+    },
+
+    /*
+     * Format the response data into a model for internal upload
+     * 1. Make this data part of the "kibana_stats" type
+     * 2. Organize the payload in the usage namespace of the data payload (usage.index, etc)
+     */
+    formatForBulkUpload: result => {
+      return {
+        type: 'kibana_stats',
```

Comment:
Should we use the constants here?

Reply:
Yeah, this can use `KIBANA_STATS_TYPE_MONITORING` from the monitoring constants.